sync_issues_to_jira/sync_to_jira.py

```diff
@@ -52,6 +53,11 @@ def main():
     with open(os.environ['GITHUB_EVENT_PATH'], 'r') as f:
         event = json.load(f)
         print(json.dumps(event, indent=4))
+    
+    # Check if it's a push event
```

Comment:
Suggest this would be easier to test and maintain as a standalone action. There isn't any conceptual relationship between this functionality and JIRA, and the script doesn't use any functions from the jira sync code.

Reply:
I plan to refactor the github-actions so it is standalone action. Unfortunately this requires much more effort than I anticipated initially and I suggest we address that in new pull request.